docker/Dockerfile.ci_lint

```
@@ -18,24 +18,24 @@
 # For lint test
 # CI docker lint env
 # tag: v0.60
-FROM ubuntu:18.04
+FROM ubuntu:22.04
 
 COPY utils/apt-install-and-clear.sh /usr/local/bin/apt-install-and-clear
 
 RUN apt-get update --fix-missing
 
 RUN apt-install-and-clear -y wget git sudo make parallel
 
-ENV TVM_VENV /venv/apache-tvm-py3.7
+ENV TVM_VENV /venv/apache-tvm-py3.8
```

```suggestion
ENV TVM_VENV /venv/apache-tvm-py3.8
ENV DEBIAN_FRONTEND noninteractive
```

we probably also need this to fix the timeout here: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-docker/detail/PR-14635/6/pipeline/77#step-212-log-1473

---

Thanks, got a fix with
``` 
RUN bash /install/ubuntu_setup_tz.sh
```